pages/contribute.md

```diff
@@ -3,24 +3,119 @@ layout: default
 title: Contribute to Auto-WCAG
 ---
 
-Auto-WCAG is a group open to anyone interested in accessibility, accessibility testing or automated testing. We welcome contributions from anyone, either though Github issues and pull requests, or by contribution to our monthly teleconferences.
+Auto-WCAG is a [W3C](https://www.w3.org/) [Community Group](https://www.w3.org/community/) open to anyone interested in accessibility or accessibility testing, manually or automated. 
```

Comment:
`interested in accessibility or accessibility testing` does not read right to me.

Reply:
Why did this change?

Reply:
@WilcoFiers, as part of the project of getting documentation that matches our current workflow and catering for new members that does not have a development background. Worked with @jkodu and @nitedog on documentation plans.

Reply:
@jkodu, fixed